packages/hyperion-autologging/src/SurfaceConsts.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates. All Rights Reserved.
+ */
+
+'use strict';
+
+export const AUTO_LOGGING_SURFACE = "data-surface";
```

Comment:
intentionally leaving off `id` suffix?

Reply:
Yes, we should have done that long ago, I put the default without `id' but of course can be changed with a config.